docs/Users/Tutorials/Linter_Bears_Advanced.rst

```diff
@@ -0,0 +1,134 @@
+Linter Bears - Advanced Feature Reference
+=========================================
+
+Often linters are no easy tools. To squeeze out the last bit of functionality
+and efficiency, ``@linter`` provides some advanced features ready for use.
+
+Supplying Configuration Files with ``generate_config``
+------------------------------------------------------
+
+Sometimes tools require a configuration file to run. ``@linter`` supports that
+easily by overriding ``generate_config()``.
+
+::
+
+    @linter(executable='...')
+    class MyBear:
+        @staticmethod
+        def generate_config(filename, file):
+            config_file = ("value1 = 1\n"
+                           "value=2 = 2")
+            return config_file
+
+The string returned from this method is written into a temporary file before
+invoking ``create_arguments()``. If you return ``None``, no configuration file
+is generated.
+
+The path of the temporary configuration file can be accessed inside
+``create_arguments()`` via the ``config_file`` parameter:
+
+::
+
+    @linter(executable='...')
+    class MyBear:
+        @staticmethod
+        def generate_config(filename, file):
+            config_file = ("value1 = 1\n"
+                           "value=2 = 2")
+            return config_file
+
+        @staticmethod
+        def create_arguments(filename, file, config_file):
+            return "--use-config", config_file
+
+.. note::
+
+    By default, no configuration file is generated.
+
+Custom Processing Functions with ``process_output``
+---------------------------------------------------
+
+Inside ``@linter`` only a few output formats are supported. And they can't be
+combined for different output streams. To specify an own output
+parsing/processing behaviour, ``process_output`` can be overridden.
```

Comment:
Like that, though the last sentence is actually wrong, this is one of the advantages that you can now combine different output formats for different output streams :)